components/KiboMobileFacetContainer.vue

```diff
@@ -36,6 +38,10 @@ export default defineComponent({
       type: Array,
       default: () => [],
     },
+    className: {
+      type: String,
+      default: "",
+    },
```

Comment:
We can set isTimeFilter: false by default, wherever required to set class , can pass true value.

Reply:
Resolved